src/components/WalletDropdown/SlideOutMenu.tsx

```diff
@@ -29,12 +18,16 @@ const Header = styled.span`
 const ClearAll = styled.div`
   display: inline-block;
   cursor: pointer;
-  margin-left: auto;
   color: ${({ theme }) => theme.accentAction};
   font-weight: 600;
   font-size: 14px;
   margin-top: auto;
   margin-bottom: auto;
+
+  :hover {
+    opacity: 0.6;
+    transition: 250ms opacity ease;
```

Comment:
I couldn't find a good style for the clear all hover state.  I reused the current hover state in the app.